src/server/models/db/schema.prisma

```diff
@@ -10,10 +11,14 @@ datasource db {
 }
 
 model List {
-  id                  Int      @id @default(autoincrement())
-  reference           String   @default(uuid()) @unique
-  createdAt           DateTime @default(now())
-  updatedAt           DateTime @updatedAt
+  id                            Int      @id @default(autoincrement())
+  reference                     String   @default(uuid()) @unique
+  createdAt                     DateTime @default(now())
+  updatedAt                     DateTime @updatedAt
+  nextAnnualReviewStartDate     DateTime @default(dbgenerated("NOW() + interval '1 year'")) @db.Date
```

Comment:
I don't think this is right. the next annualReviewStartDate is not a year from now - it's a year from the first time someone has been published. not a year from today.

I recommend make this nullable `?` - and let the scheduler fill these in every day.